package/scripts/download.ini

```diff
@@ -0,0 +1,3 @@
+[download]
+presto_rpm_url = https://s3-us-west-2.amazonaws.com/ambari-installation/presto-server-rpm-0.115t-1.x86_64.rpm
```

Comment:
its not..was more related to my above comment in metainfo.xml of switching to open source version. nvm